docs/source/ops.rst

```diff
@@ -1,7 +1,7 @@
 .. _ops:
 
-torchvision.ops
-===============
+Operators
```

Comment:
I wanted to suggest this from long time, current operators page look a bit bloated.
We should probably divide in sections.
E.g. operators on boxes, operators on masks, detection operators, etc.

Reply:
Definitely, one positive thing about the changes in this PR is that it surfaces our current doc structure weaknesses and it makes them more obvious, which will allow us to fix them.

To keep things manageable though, I would leave this out for future improvements, otherwise there will be too many things to review at once